CMakeLists.txt

```diff
@@ -598,6 +598,7 @@ endif()
 
 ## Set RCCL compile options
 target_compile_options(rccl PRIVATE -parallel-jobs=12)
+target_compile_options(rccl PRIVATE -Wuninitialized -Wsometimes-uninitialized)
```

Comment:
Furthermore, if you want to turn this kind of warnings into errors such that they can be easily identified, you can use `-Werror=uninitialized -Werror=sometimes-uninitialized`. https://godbolt.org/z/xGejP6en5

Reply:
done